pkg/util/resizefs/resizefs_linux.go

```diff
@@ -61,13 +61,19 @@ func (resizefs *ResizeFs) Resize(devicePath string) (bool, error) {
 	deviceOpened, err := resizefs.mounter.DeviceOpened(devicePath)
 
 	if err != nil {
-		deviceOpenErr := fmt.Errorf("error verifying if device %s is open: %v", devicePath, err)
+		deviceOpenErr := fmt.Errorf("ResizeFS.Resize - error verifying if device %s is open: %v", devicePath, err)
 		return false, deviceOpenErr
 	}
 
 	if deviceOpened {
-		deviceAlreadyOpenErr := fmt.Errorf("the device %s is already in use", devicePath)
-		return false, deviceAlreadyOpenErr
+		glog.Warningf("ResizeFS.Resize - Expanding mounted volume %s", devicePath)
```

Comment:
Why it's `Warning`? It should be normal operation.

Reply:
can you please fix the level? I'm inclined to approve this PR then.

Reply:
fixed